Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/LightTypesCommon.azsli

```diff
@@ -49,6 +49,6 @@ void AddSampleContribution(
     float3 intensityRgb = float3(intensity, intensity, intensity);
 
     diffuseAcc += GetDiffuseLighting(surface, lightingData, intensityRgb, posToLightSampleDir);
-    translucentAcc += GetBackLighting(surface, lightingData, intensityRgb, posToLightSampleDir, 0.0);
+    translucentAcc += GetBackLighting(surface, lightingData, intensityRgb, posToLightSampleDir, -1.0f, 0.0f);
```

Comment:
nit: explain the default parameters passed / possibly add them as const variable name for ease of understanding.

Reply:
Done in https://github.com/o3de/o3de/pull/6428/commits/69df812941fedec065e23e1965fc646f66a50624